app/config/FrontendAppConfig.scala

```diff
@@ -157,6 +166,8 @@ class FrontendAppConfigImpl @Inject() (configuration: Configuration) extends Fro
   override val baseExternalCallbackUrl: String = configuration.get[String]("urls.callback.external")
   override val baseInternalCallbackUrl: String = configuration.get[String]("urls.callback.internal")
 
+  override val fileUploadTimeoutSeconds: Long = configuration.getMillis("file-upload.timeout") / 1000
```

Comment:
AMBER: Can we specify in config and use then use the same time unit.  Either this is config specified as seconds or we change it's usage below to be in milliseconds.  100% guarentee a dev of the future will see the word seconds in `Timeout(appConfig.fileUploadTimeoutSeconds seconds)` and assume the config is in seconds.

Reply:
This is because I'm using the handy `getMillis` method defined on Configuration which under the hood reads the config value as a `Duration` and then converts to ms.